app/experimenter/experiments/forms.py

```diff
@@ -832,14 +832,14 @@ class Meta:
         fields = (
             # Required
             "review_science",
-            "review_advisory",
             "review_engineering",
             "review_qa_requested",
             "review_intent_to_ship",
             "review_bugzilla",
             "review_qa",
             "review_relman",
             # Optional
+            "review_advisory",
```

Comment:
@jaredkerim I moved "review_advisory" to optional and edited tests to reflect that since it was optional in models.py.... wasn't sure if that was intentional... could only find #745 which seems to suggest that it should be optional. Will revert if this was intentional

Reply:
@tiftran Yeah it was always optional, but there was this idea of the top sign-offs being 'ordered' in a certain sense (loosely) like you should go through them in the order they're listed.  But now that we're making a strong distinction between required and optional I think that idea goes out the window, so yes you're correct to move it to optional the way you have.